addons/rook/template/generate.sh

```diff
@@ -55,16 +55,6 @@ function generate() {
     local ceph_image=
     ceph_image="$(grep ' image: '  "${dir}/cluster/cluster.yaml" | sed -E 's/ *image: "*([^" ]+).*/\1/')"
 
-    # Upgrading passed v16.2.6 based on this note in the Rook docs
-    # https://www.rook.io/docs/rook/v1.6/ceph-upgrade.html#ceph-version-upgrades
-    # WARNING: There is a notice from Ceph for users upgrading to Ceph Pacific v16.2.6 or lower from an earlier major version of Ceph. If you are upgrading to Ceph Pacific (v16), please upgrade to v16.2.7 or higher if possible.
-    if echo "$ceph_image" | grep -q ":v16\.2" ; then
-        local ceph_version_patch="$(echo "$ceph_image" | grep -o "v[0-9]*\.[0-9]*\.[0-9]*" | sed "s/v[0-9]*\.[0-9]*\.\([0-9]*\).*/\1/")"
-        if [ "$ceph_version_patch" -lt "7" ]; then
-            ceph_image="$(echo "$ceph_image" | sed "s/v[0-9]*\.[0-9]*\.[0-9]*/v16.2.7/")"
-        fi
-    fi
-
```

Comment:
@emosbaugh should we really remove this one?

Reply:
this script is to generate new versions. the latest version of rook uses ceph quincy 17. so i dont think we will ever ship ceph pacific 16 again.